agent/pkg/dagent/utils/release.go

```diff
@@ -48,14 +49,14 @@ func DraftRelease(instance string, versionData v1.VersionData, deployResponse v1
 
 	if err != nil {
 		log.Println(fmt.Sprintln("Version drafting error ", err))
+		return
 	}
```

Comment:
```
content, err := yaml.Marshal(&release); err != nil {
		log.Println(fmt.Sprintln("Version drafting error ", err))
		return
	}
```

Refactor like this, what do you think? and @nandor-magyar

Reply:
This makes it impossible to access `content` outside of the if scope.